include/constants.hpp

```diff
@@ -89,6 +91,7 @@ constexpr auto kwdVpdInf = "com.ibm.ipzvpd.VINI";
 constexpr auto vsysInf = "com.ibm.ipzvpd.VSYS";
 constexpr auto kwdCCIN = "CC";
 constexpr auto kwdRG = "RG";
+constexpr auto kwdD0 = "D0";
```

Comment:
constants.hpp is to store any constant value which will be used in this project.
why we need to hardcode keyword name in constants.hpp? why can't it be used inplace whereever required?

Reply:
> any constant value

What is a constant value?

Reply:
I thought the team had a problem in specifying using in types. why we have constants.hpp ? @PriyangaRamasamy did not understand your concern here

Reply:
if D0 has a constant/ default value which has to be used in this project - then it sounds valid to hardcode it in constants.hpp.
why keyword name has to be stored in constants.hpp?

Reply:
> if D0 has a constant/ default value

What do you mean when you say "has a value"? I had the same question above.
In constants.hpp every lvalue "represents" a value. It does not "have a" value.
Eg: of having a value is "Variables has a value", 
here we are not defining variables, these are constexpr which are used for static time replacement to make the code more readable.

Reply:
maybe better would be to have 
constexpr auto kwdAMM = "D0";
This would help in readability

Reply:
kwdAMM = "D0"; this sounds better than kwdD0.

Reply:
Update in next PR as this keyword is being used there as well. Will get updated once it is re-based.